R/data_jessica.R

```diff
@@ -0,0 +1,32 @@
+#' The Complete History Of The NBA
+#' 
+#' This directory contains the data behind the [Complete History Of The NBA](http://fivethirtyeight.com/interactives/the-complete-history-of-every-nba-team-by-elo) interactive. Data updated periodically. Game information is from [Basketball-Reference.com](http://www.basketball-reference.com/).
```

Comment:
Url's in R help files are created using `\url{}` and not Markdown syntax. Check out `R/data_albert.R` for examples

Reply:
Also, don't forget the `@format` section. See [here](https://github.com/rudeboybert/fivethirtyeight/blob/master/R/data_albert.R) for more examples.

Reply:
Be sure to always read the help file after you've built the package to make sure the formatting looks good before commiting